component/general/materialComponents/Card2.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
```

Comment:
What's with the name of this `Card2`? What does it mean?

Reply:
I would say we can delete this one and just use Expansion Panels instead of Cards to render HNN rules.

Reply:
Let's delete the one we are not using, I think we settled on the cards we originally used in NetPyNE right?

Reply:
Card2 has been removed